src/App.tsx

```diff
@@ -1,23 +1,48 @@
-import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route, Navigate } from 'react-router-dom';
 import { Toaster } from 'react-hot-toast';
 import Layout from './components/Layout';
 import Home from './pages/Home';
 import Login from './pages/Login';
 import Register from './pages/Register';
 import BookAppointment from './pages/BookAppointment';
+import Dashboard from './pages/Dashboard';
+import { authService } from './services/api';
 
-export default function App() {
+// Protected Route component
+const ProtectedRoute = ({ children }: { children: React.ReactNode }) => {
+  const isAuthenticated = authService.isAuthenticated();
+  return isAuthenticated ? <>{children}</> : <Navigate to="/login" />;
+};
+
+function App() {
   return (
-    <BrowserRouter>
-      <Toaster position="top-right" />
+    <Router>
```

Comment:
Updated the amin App to display the features properly for new user & logged in users respectively